docs/additional-configs.md

```diff
@@ -253,16 +264,8 @@ Defaults to "ignore".
   source from where a remote Task/Pipeline definition was fetched. By default, this is set to `true`.
   To disable populating this field, set this flag to `"false"`.
 
-For example:
-
-```yaml
-apiVersion: v1
-kind: ConfigMap
-metadata:
-  name: feature-flags
-data:
-  enable-api-fields: "alpha" # Allow alpha fields to be used in Tasks and Pipelines.
-```
+- `set-security-context`: Set this flag to `true` to set a security context for containers injected by Tekton that will allow TaskRun pods
+to run in namespaces with `restricted` pod security admission.
```

Comment:
it is implied that it defaults to "false" but may be better to be explicit about it:

```suggestion
- `set-security-context`: Set this flag to `true` to set a security context for containers injected by Tekton that will allow TaskRun pods
to run in namespaces with `restricted` pod security admission. By default, this is set to `false`.
```

Reply:
done